projects/plugins/jetpack/tests/php/modules/google-analytics/test-class.jetpack-google-amp-analytics.php

```diff
@@ -36,7 +36,7 @@ public static function set_up_before_class() {
 
 		self::$woo_enabled = true;
 
-		$woo_tests_dir = __DIR__ . '/../../../../woocommerce/tests';
+		$woo_tests_dir = __DIR__ . '/../../../../../woocommerce/tests';
```

Comment:
I note the bootstrap file defines `JETPACK_WOOCOMMERCE_INSTALL_DIR`, maybe we should just use that?

Same at https://github.com/Automattic/jetpack/blob/38d1aa2ea475761eabb98666329f1feeaadd94ad/projects/plugins/jetpack/tests/php/sync/test_class.jetpack-sync-woocommerce.php#L26

Reply:
That would seem okay to me, I know @chrisbliss18 was going to be pulling over some tests changes so I won't poke it just this moment.

Reply:
I believe this whole class is outdated. `WC()->session->get( 'wc_ga_events' )` returns null and I [couldn't find any references ](https://github.com/search?q=repo%3AAutomattic%2Fwoocommerce+wc_ga_events&type=code)to `wc_ga_events` in the Woo repo 🤷